Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift

```diff
@@ -27,11 +36,21 @@ struct OfferingsList: View {
         }
         .task {
             do {
+                let offerings = try await Purchases.shared.offerings()
+                    .all
+                    .map(\.value)
+                    .sorted { $0.serverDescription > $1.serverDescription }
```

Comment:
we can delete this sorting now, right? we have it again on line 51

Reply:
That one is sorting keys (templates). This is sorting the values (offerings) by name.

Reply:
Ohh, sorry, makes sense.